ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -495,25 +497,40 @@ public class BottomSheetController: UIViewController {
     private func translateSheet(by translationDelta: CGPoint) {
         let expandedOffset = offset(for: .expanded)
         let collapsedOffset = offset(for: .collapsed)
-        let maxOffset = expandedOffset + Constants.maxRubberBandOffset
-        let minOffset = collapsedOffset - Constants.maxRubberBandOffset
+        let minOffset = expandedOffset - Constants.maxRubberBandOffset
+        let maxOffset = collapsedOffset + Constants.maxRubberBandOffset
 
         var offsetDelta = translationDelta.y
-        if currentOffsetFromBottom <= collapsedOffset || currentOffsetFromBottom >= expandedOffset {
-            offsetDelta *= translationRubberBandFactor(for: currentOffsetFromBottom)
+        if currentSheetVerticalOffset >= collapsedOffset || currentSheetVerticalOffset <= expandedOffset {
+            offsetDelta *= translationRubberBandFactor(for: currentSheetVerticalOffset)
         }
-        bottomSheetOffsetConstraint.constant = -min(max(currentOffsetFromBottom - offsetDelta, minOffset), maxOffset)
+
+        let targetOffset = min(max(bottomSheetView.frame.origin.y + offsetDelta, minOffset), maxOffset)
+        bottomSheetView.frame = sheetFrame(offset: targetOffset)
+
+        updateExpandedContentAlpha()
+        updateDimmingViewAlpha()
+    }
+
+    // Source of truth for the sheet frame at a given offset from the top of the root view bounds.
+    // The output is only meaningful once view.bounds is non-zero i.e. a layout pass has occured.
+    private func sheetFrame(offset: CGFloat) -> CGRect {
+        let availableWidth: CGFloat = view.bounds.width
+        let sheetWidth = shouldAlwaysFillWidth ? availableWidth : min(Constants.maxSheetWidth, availableWidth)
+
+        return CGRect(origin: CGPoint(x: (view.bounds.width - sheetWidth) / 2, y: offset),
+                      size: CGSize(width: sheetWidth, height: expandedSheetHeight))
     }
 
     private func translationRubberBandFactor(for currentOffset: CGFloat) -> CGFloat {
         var offLimitsOffset: CGFloat = 0.0
```

Comment:
I think we can turn the offLimitsOffset into a constant too:

```
    private func translationRubberBandFactor(for currentOffset: CGFloat) -> CGFloat {
        let offLimitsOffset: CGFloat
        let expandedOffset = offset(for: .expanded)
        let collapsedOffset = offset(for: .collapsed)

        if currentOffset < expandedOffset {
            offLimitsOffset = min(expandedOffset - currentOffset, Constants.maxRubberBandOffset)
        } else if currentOffset > collapsedOffset {
            offLimitsOffset = min(currentOffset - collapsedOffset, Constants.maxRubberBandOffset)
        } else {
            offLimitsOffset = 0
        }

        return max(1.0 - offLimitsOffset / Constants.maxRubberBandOffset, Constants.minRubberBandScaleFactor)
    }
```

Does the change make sense?